mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/controller/MockThunderheadIAMController.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.thunderhead.controller;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+@RestController
+public class MockThunderheadIAMController {
+
+    @CrossOrigin(origins = "*")
+    @PostMapping("/iam/getUser")
+    public HashMap<String, String> getUser(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
+        return new HashMap<>();
+    }
+
+    @CrossOrigin(origins = "*")
+    @PostMapping("/iam/listGroups")
+    public ArrayList<String> listGroups(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
```

Comment:
Like above, using `List` would perhaps make more sense. But this is a mock implementation (hopefully a short-lived one), so I have no strong feelings. 😃

Reply:
Unfortunately, checkstyle has a strong feeling against HashMap and ArrayList :)

Reply:
Yeah, am on it.

Reply:
thx, done